### DISCUSSION TITLE
Add mri-nufft examples

### DISCUSSION THREAD

**Discussion by matthieutrs** [deepinv/deepinv#439] [Ideas]
Suggestions for examples:

- [ ] DPIR: (already [here](https://colab.research.google.com/drive/1lsbEFn-vBMNnD61dbN4KY6tmmLmVS1VE?usp=sharing)) 
- [ ] NcPDNet

**Comment by chaithyagr** [comment]
I can help take this up.

**Comment by Andrewwango** [comment]
Please note that in #363 I propose much easier-to-use FastMRI datasets, better multi-coil MRI, models, and an example to cover all of our latest MRI functionality:

https://github.com/deepinv/deepinv/blob/74d2c780cc1c567e921f71cf00939290fe749669/examples/basics/demo_tour_mri.py

See also [this website](https://andrewwango.github.io/andrewwango/demo_tour_mri) for a rendered version.

@matthieutrs

**Comment by Andrewwango** [comment]
Can we close this @matthieutrs ? or move to discussions
